### PR TITLE
Update link to templates documentation

### LINKED ISSUE #671: Broken url in exception
Attempting to update django-compressor to be Django 1.9 compatible and ran into this error:

    compressor.exceptions.OfflineGenerationError: No template paths found. None of the configured template loaders provided template paths. See http://django.me/template-loaders for more information on template loaders.

'django.me' no longer exists. Any ideas if this was just supposed to redirect to the official Django docs on template loaders or was this to some django-compressor specific article?

### PR DESCRIPTION
**Pull request by carltongibson**
Fixes #671

### CODE REVIEW

File: compressor/management/commands/compress.py [hunk 1]
```diff
@@ -148,7 +148,7 @@ def compress(self, log=None, **options):
             raise OfflineGenerationError("No template paths found. None of "
                                          "the configured template loaders "
                                          "provided template paths. See "
-                                         "http://django.me/template-loaders "
+                                         "https://docs.djangoproject.com/en/1.8/topics/templates/ "
```
Comment (diox) [inline compressor/management/commands/compress.py]: Should we use that link, or https://docs.djangoproject.com/en/1.8/ref/templates/api/#loading-templates ? or https://docs.djangoproject.com/en/1.8/ref/templates/api/#loader-types ? Up to you.

Comment (carltongibson) [inline compressor/management/commands/compress.py]: Not sure... I initial thought about linking there — but with 1.8's new features I thought the more general link...

Comment (diox) [inline compressor/management/commands/compress.py]: General link works for me

Comment (carltongibson) [inline compressor/management/commands/compress.py]: OK. Will merge as is.

**Pull request merged**